react/week4/git-users/src/UserContext.js

```diff
@@ -1,22 +1,32 @@
 import React, { useState, useEffect, createContext } from "react";
 
 export const userContext = createContext();
-// const defaultState = {
-//   users: "",
-//   loading: false,
-// };
+
 const UserProvider = (props) => {
```

Comment:
Change `const UserProvider = (props)` -> `const UserProvider = ({children})` because you only use children in your implementation. It's cleaner and tell the person who check the code that is the only prop being used

Reply:
done, thanks.